cmd/build.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2017 The Kedge Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kedgeproject/kedge/pkg/build"
+
+	"github.com/spf13/cobra"
+)
+
+var Dockerfile string
+var DockerImage string
+var DockerContext string
+var PushImage bool
+
+var buildCmd = &cobra.Command{
+	Use:   "build",
+	Short: "build application image",
+	Run: func(cmd *cobra.Command, args []string) {
+		if DockerImage == "" {
+			fmt.Println("Please specify the container image name using flag '--image' or '-i'")
+			os.Exit(-1)
+		}
+		if err := build.BuildPushDockerImage(Dockerfile, DockerImage, DockerContext, PushImage); err != nil {
+			fmt.Println(err)
+			os.Exit(-1)
+		}
+	},
+}
+
+func init() {
+
+	buildCmd.Flags().StringVarP(&Dockerfile, "file", "f", "Dockerfile", "Specify Dockerfile for doing builds")
+	buildCmd.Flags().StringVarP(&DockerImage, "image", "i", "", "Name of docker image to be built")
+	buildCmd.Flags().StringVarP(&DockerContext, "context", "c", ".", "Context is path relative to which the Dockerfile paths will be set")
```

Comment:
Wouldn't it be better to have information about relatives of Dockerfile be written in help for `--file`, rather than here? The whole sentence sounds a bit confusing to me.

Reply:
not sure what you mean here! :(